Fabric.Identity.API/Fabric.Identity.API.csproj

```diff
@@ -1,23 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
     <DockerComposeProjectPath>..\docker-compose.dcproj</DockerComposeProjectPath>
     <Company>Health Catalyst</Company>
     <Product>Fabric.Identity</Product>
     <Copyright>Copyright © 2018</Copyright>
     <Version Condition=" '$(BUILD_BUILDNUMBER)' == '' ">1.0.0.0-local</Version>
     <Version Condition=" '$(BUILD_BUILDNUMBER)' != '' ">$(BUILD_BUILDNUMBER)</Version>
+    <CopyRefAssembliesToPublishDirectory>true</CopyRefAssembliesToPublishDirectory>
+    <AspNetCoreHostingModel>OutOfProcess</AspNetCoreHostingModel>
```

Comment:
Why are we choosing OutOfProcess instead of InProcess for hosting in Asp.Net Core 2.2. Is it because we want to be compatible with Linux as well so InProcess won't work for linux? InProcess provides better performance and is less resource intensive. There isn't an extra network hop between IIS and Kestral or maintaining an additional process.